### ISSUE TITLE
split stem notation

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/music-encoding#721]
How to encode split stem notation? 

![split stem](https://steinberg.help/dorico/v2/en/_shared_picts/picts/dorico/notation_reference/accidentals_altered_unison_tree.png)

For "easy" cases like altered unions in chords you can avoid split stem notation. [Dorico](https://steinberg.help/dorico/v2/en/dorico/topics/notation_reference/notation_reference_stems_altered_unisons_split_stems_c.html) lets you choose the display of these cases. 

But there are more complicated examples like in Bartók's Violin Sonata (with a triplet unison)
![split-stem](https://user-images.githubusercontent.com/7693447/96251519-e5dc1a80-0fb0-11eb-8d96-d70f7eea6c2a.png)

Could the proposed `@stem.sameas` in #586 be used for this somehow or do we need even more control like relative positions of the noteheads?

**Comment by rettinghaus** [comment]
If we assume a stem of a chord is vertical (i.e., right angle to the staff lines), it should be sufficient to tell that a note of that very chord is moved left or right from the main stem. So I propose to add a `stem.split` attribute that could hold the values `left`, `right` and `none`.

**Comment by lpugin** [comment]
I think  `left` and `right` would be enough. (The absence of attribute value would mean `none`.)

**Comment by ahankinson** [comment]
How would we know which pitches are attached to the split stem?

**Comment by rettinghaus** [comment]
Something like:

```xml
<chord>
  <note pname="c" stem.split="left" />
  <note pname="d" />
  <note pname="e" stem.split="right" />
</chord>
```

**Comment by ahankinson** [comment]
So I guess the documentation for split stem **chords** should specify that the stem splitting is captured on the individual notes on the chords, not on the chord itself. 

Otherwise it would be tempting to write

```
<chord stem.split="right">
   ...
</chord>
```

Which would not work. We should also make sure that chords don't get this attribute, since it wouldn't make sense.

**Comment by lpugin** [comment]
Another proposal would be to add the values `splitLeft` and `splitRight` to data.STEMPOSITON (https://music-encoding.org/guidelines/dev/data-types/data.STEMPOSITION.html)
That way we do not need a new attribute. Of course, controlling that this  is not happening on a chord would still need to be controlled via Schematron if desired.

**Comment by ahankinson** [comment]
Although I like that, it does cause me to wonder about the differences between "pos=right" and "pos=splitRight". The first means it's attached to the right side of the note, while the second means the stem is split to the right. This would make it not possible to split a stem and attach the stem to a different side of the note head...

**Comment by pe-ro** [comment]
@ahankinson , did you mean to say "This would make it possible to split a stem and attach the stem to a different side of the note head"?  If so, I think the answer is "yes", but we could control this contradiction with Schematron or we can allow the renderer to check for "impossibilities".  @lpugin ?

**Comment by lpugin** [comment]
> This would make it not possible to split a stem and attach the stem to a different side of the note head...

Yes, this is correct. But not being a specialist in split-stem-notation I cannot really judge if this is really an issue or not ;-)

**Comment by ahankinson** [comment]
Well, just looking at the examples, I'm not really sure where on the note head the stem would be attached. 

@pe-ro no, I was meaning that we are overloading the `@pos`  attribute with things that have slightly different functions. The first is where the stem is attached; the second is how the stem (from some parent?) is behaving. Would the second example above be:

`<note stem.pos="left splitRight">`

Can `@pos` take more than one token?

**Comment by lpugin** [comment]
> Well, just looking at the examples, I'm not really sure where on the note head the stem would be attached.

From the examples I have seen it seems always to be attached to the side further apart. But of course, this does not mean it is a rule.

> `<note stem.pos="left splitRight">`

I think if we need that, having two attributes `stem.pos` and `stem.split` as suggested before would better.

**Comment by pe-ro** [comment]
Yes, having 2 attributes is best.

**Comment by bwbohl** [comment]
- [x] modify schema
- [ ] add example to the guidelines
